File: punto4.c
#include <stdio.h>
#include <time.h>

int main() {
    long long suma = 0;
    clock_t inicio = clock();

    for (long long i = 1; i <= 100000000; i++) {
        suma += i;
    }

    clock_t fin = clock();
    double tiempo = (double)(fin - inicio) / CLOCKS_PER_SEC;

    printf("Resultado: %lld\n", suma);
    printf("Tiempo de ejecucion en C: %f segundos\n", tiempo);

    return 0;
}

File: punto4.py
import time

suma = 0
inicio = time.time()

for i in range(1, 100000001):
    suma += i

fin = time.time()
tiempo = fin - inicio

print(f"Resultado: {suma}")
print(f"Tiempo de ejecucion en Python: {tiempo:.6f} segundos")

